Frontend/ui-customer/src/Helpers/Storage/Storage.purs

```diff
@@ -89,6 +89,7 @@ data KeyStore
   | LAST_KNOWN_LAT
   | LAST_KNOWN_LON
   | SOURCE_GEOHASHES
+  | REALLOCATE_PRODUCT_ENABLED
```

Comment:
Why this localStorage variable required ?

Reply:
to enable from native

Reply:
to enable from native as in? If this is required in native as well can we get this config from remote config? instead of LOCAL_STORAGE?

Reply:
`REALLOCATE_PRODUCT_ENABLED` was added to android native 6-7 months ago and was set to `false` so that we don't receive reallocation notification